dist/api/bot.js

```diff
@@ -0,0 +1,65 @@
+"use strict";
```

Comment:
any reason to add `dist/`?

Reply:
Willie and I can't compile TS in Compute Engine for some reason, so I added `dist/` so that I can build locally and use that build in CE. I'm thinking we can dockerize the bot so that we don't have to worry about physical files in CE and we can just use k8s or something like that.

Reply:
i thought that was the issue.. its because the vm we have is too underpowered and hangs when building. I still recommend not adding this to git because we'll just end up with a lot of commits like "build to dist", instead should use sftp imo. Up to you if you want to change this now or later tho

Reply:
It will definitely change when it's dockerized or when compute engine decides to start working; 'twas only for testing so far. Ideally, the source and any dependency specifications are the only things added to the repo, so before we merge it'll fix.